bedrock/firefox/templates/firefox/developer.html

```diff
@@ -93,6 +93,28 @@
 <section class="feature-overview">
   <div class="container">
     <ul class="features">
+    {% if LANG.startswith('en-') %}
```

Comment:
The page is fully localized, preventing our localizers to work on it after Nov 3rd seems wrong, unless the blurbs are disappearing after a few days.